webapp/public/js/domjudge.js

```diff
@@ -575,9 +572,9 @@ function updateMenuJudgehosts(data)
         for(var i=0; i<num; i++) {
             sendNotification('Judgehost down.',
                 {'tag': 'host_'+data[i].hostname+'@'+
-                 Math.floor(data[i].polltime),
-                 'link': domjudge_base_url + '/jury/judgehosts/' + encodeURIComponent(data[i].hostname),
-                 'body': data[i].hostname + ' is down'});
+                    Math.floor(data[i].polltime),
+                    'link': domjudge_base_url + '/jury/judgehosts/' + encodeURIComponent(data[i].hostname),
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mvr320/domjudge/pullRequest?prid=8427924)
```suggestion
                 "link": domjudge_base_url + '/jury/judgehosts/' + encodeURIComponent(data[i].hostname),
```